pytest_fixtures/component/taxonomy.py

```diff
@@ -52,6 +52,11 @@ def function_location():
     return entities.Location().create()
 
 
+@pytest.fixture(scope='function')
+def function_location_with_org(default_sat):
+    return default_sat.api.Location(organization=[function_org]).create()
```

Comment:
I think `function_org` should be also in the parameters of this fixture

Reply:
nice catch, I believe this is the only catch related to the test failures made by these changes :+1: